Gopkg.toml

```diff
@@ -0,0 +1,86 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  branch = "master"
+  name = "code.cloudfoundry.org/lager"
+
+[[constraint]]
+  name = "github.com/go-sql-driver/mysql"
+  version = "~1.3.0"
```

Comment:
Dep imported the existing glide file and resolve all the dependencies so they worked together while trying to maintain the highest versions.

My intuition says we should always use master for `/x/` golang stuff so we can stay up to date with the most recent standard library and wherever possible use `^` i.e. in libraries we use.

I could see the benefit of hard-fixing or severely limiting something like `github.com/pivotal-cf/brokerapi` because them adding features means we might no longer satisfy their interface.

Thoughts?

Reply:
That sounds good to me